libs/pipingsidesheet/src/lib/utils-sidesheet/useGetInsulationTags.ts

```diff
@@ -0,0 +1,29 @@
+import { useHttpClient } from '@equinor/fusion-framework-react-app/http';
+import { useContextId } from '@cc-components/shared';
+import { useQuery } from '@tanstack/react-query';
+import { InsulationTagResponse } from '@cc-components/pipingshared';
+
+export const useGetInsulationTags = (pipetestId: string) => {
+  const client = useHttpClient('cc-api');
```

Comment:
using the useHttpClient hook from @cc-components/shared is generally better for consuming the "main" client. Its a simple abstraction to prevent the "cc-api" values from cluttering the codebase

Reply:
Fixed!